trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDeocrator < Decorator
```

Comment:
[OPTIONAL] I kindly suggest that you correct the spelling of TrimmerDeocrator to TrimmerDecorator. It could cause issues and make your code not work.